### PR TITLE
Feature/uppsf 3490 upgrade to msk

### PR DESCRIPTION
**Pull request by martin-stanchev**
# Description

## What

Migrating to MSK and upgrading the service

## Why

https://financialtimes.atlassian.net/browse/UPPSF-3490

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [x] Feature
- [ ] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

File: helm/native-ingester/app-configs/native-ingester-cms_eks_publish.yaml [hunk 1]
```diff
@@ -3,12 +3,13 @@ replicaCount: 1
...
-  QUEUE_READ_GROUP: "nativeIngesterCms"
-  QUEUE_READ_TOPIC: "PreNativeCmsPublicationEvents"
+  CONSUMER_GROUP: "native-ingester-cms"
+  CONSUMER_TOPIC: "PreNativeCmsPublicationEvents"
   NATIVE_CONTENT_UUID_FIELDS: "uuid,post.uuid,data.uuidv3,id"
   CONFIG: "config.json"
-  QUEUE_WRITE_TOPIC: "NativeCmsPublicationEvents"
+  PRODUCER_TOPIC: "NativeCmsPublicationEvents"
   PANIC_GUIDE_URL: "https://runbooks.in.ft.com/native-ingester"
+  KAFKA_LAG_TOLERANCE: 500
```
Comment (martin-stanchev) [inline helm/native-ingester/app-configs/native-ingester-cms_eks_publish.yaml:12]: I am not sure if this is too high, but I was concerned if the service will lag when some large batches of republishes go through.

File: go.mod [hunk 2]
```diff
@@ -1,31 +1,43 @@
 module github.com/Financial-Times/native-ingester
 
-go 1.13
+go 1.18
```
Comment (ilian2233) [inline go.mod:3]: We can start using 1.19 if everyone is ok with it.

**Review by ilian2233 — APPROVED** [review]

**Review by ilian2233 — DISMISSED** [review]

File: native/native_writer.go [hunk 3]
```diff
@@ -80,7 +84,7 @@ func (nw *nativeWriter) WriteToCollection(msg NativeMessage, collection string)
 	}
 
 	if request.Header.Get(contentTypeHeader) == "" {
-		logger.NewEntry(msg.TransactionID()).
+		log.
 			WithUUID(contentUUID).
```
Comment (dtvalk-ov) [inline native/native_writer.go:88]: .WithUUID line can be removed as well

File: queue/message_handler.go [hunk 4]
```diff
@@ -57,25 +63,24 @@ func (mh *MessageHandler) HandleMessage(msg kafka.FTMessage) error {
...
-			logger.NewMonitoringEntry("Ingest", pubEvent.transactionID(), mh.contentType).
+			mh.logger.WithMonitoringEvent("Ingest", pubEvent.transactionID(), mh.contentType).
 				WithUUID(contentUUID).
 				WithError(forwardErr).
 				Error("Failed to forward consumed message to a different queue")
-			return forwardErr
+			return
 		}
-		logger.NewMonitoringEntry("Ingest", pubEvent.transactionID(), mh.contentType).
+		mh.logger.WithMonitoringEvent("Ingest", pubEvent.transactionID(), mh.contentType).
```
Comment (dtvalk-ov) [inline queue/message_handler.go:75]: I guess we could extract something like
log := mh.logger.WithMonitoringEvent("Ingest", pubEvent.transactionID(), mh.contentType) above all the checks that produce that monitoring event and just use the log entry to print the specific error and message

**Review by dtvalk-ov — APPROVED** [review]

**Pull request merged**